### DISCUSSION TITLE
Aggregated dashboard with stats for all sites

### DISCUSSION THREAD

**Discussion by lumpysimon** [plausible/analytics#724] [Anything Else]
Is a dashboard that contains aggregated high level data for all sites in an account something you have considered?

It probably wouldn't need as much detail as the individual sites' statistics provide, and no drill-down, but would be great to be able to easily see total pageviews, visitors, top pages, etc.

**Comment by metmarkosaric** [comment]
Thanks @lumpysimon for the suggestion! When we recently redesigned the /sites page, we considered how to give a quick overview of all sites and ended up with the "last 24 hour" metric being displayed for all. It quickly tells you what's happened in the last day for individual sites and you can then click to dig deeper in case something looks interesting.

In general we're very mindful of the loading time of the dashboard. Introducing more aggregated metrics will slow down the loading time dramatically for many of our users. Many have 20+ sites in the same account, some have tens of millions of page views... Aggregating all this will take a while to load so there are no current plans to do it but we're keeping the option open for the future.

**Reply by lumpysimon** [reply]
Thank you for your very quick and detailed reply Marko, I really appreciate that :) All makes sense, and I definitely agree that a slow dashboard would be very unwanted.

Thinking about it, I guess I could use the API to request the data for each site, then do the totalling at my end. I'll wait until the API is more stable than have a play around with it.
